src/routes/index.svelte

```diff
@@ -39,7 +43,7 @@
 <h1>Great success!</h1>
 
 <figure>
-	<img alt='Success Kid' src='successkid.jpg'>
+	<img alt='Success Kid' src='{successkid}'>
```

Comment:
The quotes should be removed here, and the alt quotes turned to doublequotes.

Reply:
I agree about the double quotes, so changed them both to that. Not sure about removing the quotes since I believe that wouldn't be valid html and could fail for certain urls (not this one though)